application/Bootstrap.php

```diff
@@ -141,6 +147,8 @@ private function is_plugin_active( $plugin ) {
 				return ( apply_filters( 'toolset_is_wpml_active_and_configured', false ) );
 			case 'toolset':
 				return apply_filters( 'types_is_active', false ) || $this->is_plugin_active( 'views' );
+			case 'layouts':
+				return true ; // FIXME: do we need a real check here?
```

Comment:
Yes please, this will prevent any attempt to interact with Layouts if the plugin is not active. Advantage: All future subcommands of the `layouts` command don't have to worry about that anymore.

You can use this:

```suggestion
				return defined( 'WPDDL_DEVELOPMENT' ) || defined( 'WPDDL_PRODUCTION' );
```